test/e2e/app-dir/app-routes/helpers.ts

```diff
@@ -63,6 +63,7 @@ export function getRequestMeta(
     | Cookies
     | ReadonlyHeaders
     | ReadonlyRequestCookies
+    // @ts-ignore
```

Comment:
Why we need to ignore this one? If we have to workaround can we use `ts-expect-error` with some explanation?